### PR TITLE
Update manifest.json

### PR DESCRIPTION
**Pull request by minepodcraft**
## Type of change

- [ ] Add new directory for a server
- [x] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [ ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ ] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/onthepixel/manifest.json [hunk 1]
```diff
@@ -20,20 +20,20 @@
   },
   "gamemodes": {
     "kitpvp": {
```
Comment (DoJapHD) [inline minecraft_servers/onthepixel/manifest.json:22]: Change key if you change its purpose

File: minecraft_servers/onthepixel/manifest.json [hunk 2]
```diff
@@ -20,20 +20,20 @@
...
-      "command": "/kpvp"
+      "name": "Hide and Seek",
+      "command": "/hideandseek"
     },
     "bedwards":{
       "name": "BedWars",
-      "command": "/bw"
+      "command": "/bedwars"
     },
     "tntrun":{
```
Comment (DoJapHD) [inline minecraft_servers/onthepixel/manifest.json:30]: Same here

File: minecraft_servers/onthepixel/manifest.json [hunk 3]
```diff
@@ -20,20 +20,20 @@
...
-      "name": "TNTRun",
-      "command": "/TNTRun"
+      "name": "BuildFFA",
+      "command": "/buildffa"
     },
     "bridging":{
```
Comment (DoJapHD) [inline minecraft_servers/onthepixel/manifest.json:34]: Same here

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Why is the server under maintenance?

**Comment by minepodcraft** [comment]
> Why is the server under maintenance?

we are rebuilding the server

**Comment by DoJapHD** [comment]
Oh well, normally only released servers are getting added to the repository but as it's already added, I'll make an exception

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**